src/pre_process/m_mpi_proxy.fpp

```diff
@@ -91,6 +91,7 @@ contains
                 call MPI_BCAST(patch_icpp(i)%${VAR}$, size(patch_icpp(i)%${VAR}$), MPI_DOUBLE_PRECISION, 0, MPI_COMM_WORLD, ierr)
             #:endfor
 
+            call MPI_BCAST(patch_icpp(i)%model%threshold, 1, MPI_DOUBLE_PRECISION, 0, MPI_COMM_WORLD, ierr)
```

Comment:
I believe this could be added to the fypp loop above

Reply:
This new input is not an array unlike others in the loop. I am not sure how to add it.

Reply:
yes